cmd/kogito/command/project/new_project.go

```diff
@@ -74,6 +76,7 @@ func (i *newProjectCommand) InitHook() {
 	i.flags = newProjectFlags{}
 	i.Parent.AddCommand(i.command)
 	i.command.Flags().StringVarP(&i.flags.project, "project", "n", "", "The project project")
+	i.command.Flags().BoolVar(&i.flags.installDataIndex, "install-data-index", true, "Installs the default instance of Data Index being provisioned by the Kogito Operator in the new project")
```

Comment:
'true' means it will be installed by default ?

Reply:
You and @mswiderski could start a fight regarding this flag :p
I just want to merge this PR by the end of the day to have time to push it to 0.6.0 :)